adapter/pkg/logging/log_constants.go

```diff
@@ -29,3 +29,25 @@ const (
 	infoLevel       = "INFO"
 	debugLevel      = "DEBG"
 )
+
+// Log Formalization Constants
+const (
+	TEXT = "TEXT"
+	JSON = "JSON"
+)
+
+// Log (Error) severity level constants
+const (
+	BLOCKER  = "Blocker"
+	CRITICAL = "Critical"
+	MAJOR    = "Major"
+	MINOR    = "Minor"
+	TRIVIAL  = "Trivial"
+	DEFAULT  = "Default"
+)
+
+// Error Log attribute name constants
+const (
+	SEVERITY  = "severity"
+	ErrorCode = "error_code"
```

Comment:
Shall we rename ErrorCode as well. (with uppercase)

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/2633/commits/bcc6c01f44c2ed99de9bd6699593d8160ace82b6